### ISSUE TITLE
Have we utilized class information in "submission-trained"

### ISSUE THREAD

**Issue by akhanss** [ducha-aiki/google-retrieval-challenge-2019-fastai-starter#1]
Hi @ducha-aiki 
Thank you for your great repo and sharing the code. I am just wondering whether we utilized class information of index set (essentially, we don't know ground truth of this index set) for the retrieval task. If yes, how it works, specifically, in submission-trained.ipynb notebook. Thanks in advance.

**Comment by ducha-aiki** [comment]
We don't use competition data for training at all here

**Comment by akhanss** [comment]
@ducha-aiki Thank you for your kind and prompt reply.
Yes, I know you didn't use competition data, however, perhaps you utilized oxford5k/paris6k data.

I am concerned about the following code snippet in the submission-train.ipynb notebook.

#Now we will load training network and transfer its weights to the inference net
**TRAIN_CLASSES=780** #that is hardcoded from number of classes in training dataset
learn = Learner(data, RingGeMNet(models.densenet121(pretrained=True), TRAIN_CLASSES),
                   metrics=[accuracy],
                   loss_func=nn.CrossEntropyLoss())

InferenceNet =  GeMNet(models.densenet121())
**InferenceNet.cnn.load_state_dict(learn.model.cnn.state_dict())**

**Comment by ducha-aiki** [comment]
Inference Net has no last clf layer, just features
